library/init/meta/interactive.lean

```diff
@@ -66,6 +81,7 @@ private meta def parser_desc_aux : expr → tactic (list format)
 | `(ident)  := return ["id"]
 | `(ident_) := return ["id"]
 | `(qexpr) := return ["expr"]
+| `(location) := return ["loc?"]
```

Comment:
Did you try the default description? I can see that it may not be quite readable.